blocks.js

```diff
@@ -7554,24 +7556,26 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-        // } else if (key.indexOf('§_def') === 0) { 
-        // menu.addItem(choices[key].blockInstance(), choices[key]);
-            } else if (key.charCodeAt(key.length - 1) == 0x25ba) {
+         // } else if (key.indexOf('§_def') === 0) { 
+         //     menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charAt(key.length - 1) == '{') {
```

Comment:
I see a lot in the new code that was written, I can point them about, but maybe I should just setup an appropriate tool to do it. @snap-bot will be happy to comment when he's ready. 😉 

(Also the unicode character you're using can totally be embedded in our file, but it's probably safer to actually refer to the value, rather than use the actual glyph.)